### DISCUSSION TITLE
Import cv2 as cv

### DISCUSSION THREAD

**Discussion by Blaise-wig** [jyjblrd/Mocap-Drones#22] [General]
Thank you for sharing this amazing project. 
I installed opencv with the extra modules with cmake. When I import cv2 as cv as in your code, an error pops up (No module named 'cv2'). Do I have to specify the path where the opencv is installed? if yes, How? or is there another way?

**Comment by NightHawk32** [comment]
Did you check your build folder for the python loader? You need to run the setup.py script which is generated in there.

**Reply by cultist123** [reply]
> Did you check your build folder for the python loader? You need to run the setup.py script which is generated in there.

I didn't understand what you meant. Could you please explain it in more detail?

**Comment by cultist123** [comment]
> Thank you for sharing this amazing project. I installed opencv with the extra modules with cmake. When I import cv2 as cv as in your code, an error pops up (No module named 'cv2'). Do I have to specify the path where the opencv is installed? if yes, How? or is there another way?

I encountered the same problem as you. How did you eventually solve it?
